client/blocks/reader-combined-card/index.jsx

```diff
@@ -59,18 +60,27 @@ class ReaderCombinedCard extends React.Component {
 			feed_id: postKey.feedId,
 			post_count: size( posts ),
 		} );
-	}
+	};
 
 	render() {
-		const { posts, site, feed, postKey, selectedPostKey, onClick, isDiscover, translate } = this.props;
+		const {
+			posts,
+			site,
+			feed,
+			postKey,
+			selectedPostKey,
+			onClick,
+			isDiscover,
+			translate,
+		} = this.props;
 		const feedId = postKey.feedId;
 		const siteId = postKey.blogId;
 		const siteIcon = get( site, 'icon.img' );
 		const feedIcon = get( feed, 'image' );
 		const streamUrl = getStreamUrl( feedId, siteId );
 		const siteName = getSiteName( { site, post: posts[ 0 ] } );
 		const isSelectedPost = post => keysAreEqual( keyForPost( post ), selectedPostKey );
-		const followUrl = feed && feed.URL || site && site.URL;
+		const followUrl = ( feed && feed.URL ) || ( site && site.URL );
```

Comment:
also, it's wrong. :) 

Should be looking for feed_URL on the site first. Switch to using the helper?